addons/purchase/views/portal_templates.xml

```diff
@@ -374,7 +374,7 @@
   <template id="purchase_order_portal_content_totals_table">
       <table class="table table-sm">
           <t t-set="tax_totals" t-value="order.tax_totals"/>
-          <t t-call="account.document_tax_totals"/>
+          <t t-call="purchase.document_tax_totals"/>
```

Comment:
Thinking of it... Are we sure we want the fields added on studio to be on the portal too? :thinking: 
But then, if we change it in studio while still calling the initial template, changes will be applied on every report too, which we certainly don't want either... Maybe ask @dalaOdoo what he thought of it, I'm not totally sure what can/can't be done in studio... :thinking:

Reply:
@Demesmaeker I asked, and from what I understand it is supposed to be like this, that when you change in the report it also changes in portal view.